apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -63,18 +69,27 @@ class ItineraryCard extends React.Component<Props> {
     };
     return (
       <View style={styles.card}>
-        <View>
+        {Platform.OS === 'web' && (
+          <Text style={styles.price}>{formatPrice(priceObject)}</Text>
```

Comment:
Should we instead change the `LocalizedPrice` component in @kiwicom/universal-components?

Reply:
I've planned to replace it once the `LocalizedPrice` web style is updated. But you are right, the current design is not that different so I replaced it with `LocalizedPrice` component and will update its web styles in `universal-components`.